docs/src/tutorial/clean_shutdown.md

```diff
@@ -60,7 +60,7 @@ async fn broker(mut events: Receiver<Event>) -> Result<()> {
                     Entry::Vacant(entry) => {
                         let (client_sender, client_receiver) = mpsc::unbounded();
                         entry.insert(client_sender);
-                        let handle = spawn_and_log_error(client_writer(client_receiver, stream));
+                        let handle = client_writer(client_receiver, stream);
```

Comment:
I think it'd be best to still spawn the task here, otherwise futures will not execute in parallel.

If we just collect these futures into a `Vec` and then await them at line 72, they will execute serially. But if we spawn them first, they will execute in the background on other threads, and at line 72 we just await these tasks to complete.